velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -490,7 +490,15 @@ VectorPtr VectorFuzzer::fuzzFlat(const TypePtr& type, vector_size_t size) {
     for (const auto& childType : rowType.children()) {
       childrenVectors.emplace_back(fuzzFlat(childType, size));
     }
-    return fuzzRow(std::move(childrenVectors), size);
+
+    std::vector<std::string> childrenNames;
+    for (auto& childName : rowType.names()) {
+      childrenNames.emplace_back(childName);
+    }
+    while (childrenNames.size() < rowType.children().size()) {
+      childrenNames.emplace_back("");
+    }
+    return fuzzRow(std::move(childrenVectors), std::move(childrenNames), size);
```

Comment:
could we simply do:

`return fuzzRow(std::move(childrenVectors), rowType.names(), size);`

?